deploy/helm/kubecf/assets/operations/instance_groups/database.yaml

```diff
@@ -1,3 +1,323 @@
+- type: remove
+  path: /variables/name=routing_api_database_password
+- type: remove
+  path: /variables/name=locket_database_password
+- type: remove
+  path: /variables/name=cc_database_password
+- type: remove
+  path: /variables/name=credhub_database_password
+- type: remove
+  path: /variables/name=diego_database_password
+- type: remove
+  path: /variables/name=uaa_database_password
+- type: remove
+  path: /variables/name=pxc_galera_ca
+- type: remove
+  path: /variables/name=pxc_server_ca
+- type: remove
+  path: /variables/name=galera_server_certificate
+- type: remove
+  path: /variables/name=mysql_server_certificate
+
+# Remove database links.
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/consumes?
+  value: {database: nil}
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/consumes?
+  value: {database: nil}
+- type: replace
+  path: /instance_groups/name=cc-worker/jobs/name=cloud_controller_worker/consumes?
+  value: {database: nil}
+- type: replace
+  path: /instance_groups/name=scheduler/jobs/name=cloud_controller_clock/consumes?
+  value: {database: nil}
+- type: replace
+  path: /instance_groups/name=scheduler/jobs/name=cc_deployment_updater/consumes?
+  value: {database: nil}
+- type: replace
+  path: /instance_groups/name=diego-api/jobs/name=silk-controller/consumes?
+  value: {database: nil}
+- type: replace
+  path: /instance_groups/name=api/jobs/name=policy-server/consumes?
+  value: {database: nil}
+
+# Replace database config for the jobs.
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/db_scheme
+  value: {{ .Values.features.external_database.type | quote }}
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/port
+  value: {{ .Values.features.external_database.port }}
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/databases/tag=uaa/name
+  value: {{ .Values.features.external_database.databases.uaa.name | quote }}
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/address?
+  value: {{ .Values.features.external_database.host | quote }}
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/roles/name=uaa/password
+  value: {{ .Values.features.external_database.databases.uaa.password | quote }}
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/roles/name=uaa/name
+  value: {{ .Values.features.external_database.databases.uaa.username | quote }}
+- type: remove
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaa?/ca_certs
+- type: remove
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/tls_enabled?
+
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/ccdb/db_scheme
+  value: &external_cc_database_scheme {{ .Values.features.external_database.type | quote }}
```

Comment:
Would it make sense to replace all of `…/name=cloud_controller_ng/properties/ccdb` as one block?

Reply:
It'd be more complicated to handle the `.../properties/ccdb/databases/tag=cc/name`.

Reply:
I assume we don't want to completely replace this.